### PR TITLE
Fixed celebration template Logo of Layer5 with Meshery

### PR DESCRIPTION
**Pull request by yash37158**
**Description**

This PR fixes #
This PR fixes the existing layer5 celebration logo with the meshery celebration as meshery-kuma is under the organization meshery
**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by yash37158** [comment]
Hey @leecalcote any comments on this ?

**Comment by leecalcote** [comment]
@yash37158  please engage another maintainer. I will only slow you down.

In the meantime, you can amend your commit(s) with a signoff to pass DCO.

**Comment by yash37158** [comment]
Sure @leecalcote

File: .github/config.yml [hunk 1]
```diff
@@ -15,9 +15,13 @@ newPRWelcomeComment: >
 # Configuration for first-pr-merge - https://github.com/behaviorbot/first-pr-merge
 # Comment to be posted to on pull requests merged by a first time user
 firstPRMergeComment: >
-  Thanks for your contribution to the Layer5 community! :tada:
+  Thanks for your contribution to the Layer5 and Meshery community! :tada:
```
Comment (Chadha93) [inline .github/config.yml:18]:
```suggestion
  Thanks for your contribution to Meshery! :tada:
```

File: .github/config.yml [hunk 2]
```diff
@@ -15,9 +15,13 @@ newPRWelcomeComment: >
...
    
-  ![Congrats!](https://raw.githubusercontent.com/meshery/meshery.io/master/.github/welcome/Layer5-celebration.png)
+  {% if site.github.dark_mode %}
+  ![Congrats!](https://raw.githubusercontent.com/meshery/meshery.io/master/.github/welcome/meshery-celebration-dark.svg)
+  {% else %}
+  ![Congrats!](https://raw.githubusercontent.com/meshery/meshery.io/master/.github/welcome/meshery-celebration-light.svg)
```
Comment (thisiskaransgit) [inline .github/config.yml:23]: Update the links with `meshery-kuma`'s path
https://github.com/meshery/meshery-kuma/blob/master/.github/welcome/meshery-celebration-dark.svg
https://github.com/meshery/meshery-kuma/blob/master/.github/welcome/meshery-celebration-light.svg

**Review by thisiskaransgit — CHANGES_REQUESTED** [review]

**Comment by thisiskaransgit** [comment]
Mention the fixed issue in description.

**Pull request closed without merging**